terraform/helpers/build

```
@@ -17,6 +17,6 @@ os="$(uname -s | tr '[:upper:]' '[:lower:]')"
 hcl2json_checksum="24068f1e25a34d8f8ca763f34fce11527472891bfa834d1504f665855021d5d4"
 hcl2json_url="https://github.com/tmccombs/hcl2json/releases/download/v0.3.3/hcl2json_${os}_amd64"
 hcl2json_path="$install_dir/bin/hcl2json"
-wget -O "$hcl2json_path" "$hcl2json_url"
+curl -LsSo "$hcl2json_path" "$hcl2json_url"
```

Could this use the same options as the others?
```suggestion
curl -sSLfo "$hcl2json_path" "$hcl2json_url"
```

---

sure, it's done.